src/mocks/handlers.js

```diff
@@ -0,0 +1,195 @@
+/* eslint import/no-extraneous-dependencies:off */
+import { rest } from 'msw';
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.